### PR TITLE
Issue #66 fix integration tests

### LINKED ISSUE #66: fix integration tests

### PR DESCRIPTION
**Pull request by crookse**
Fixes #66

**Description**

* Fixes and restructures the integration tests to work with the new syntax
* Removes the browser console apps since the documentation will go more in depth about building and using a chat app

### CODE REVIEW

File: tests/integration/app_3000/tests.ts [hunk 1]
```diff
@@ -42,118 +27,108 @@ const webServer = new Drash.Http.Server({
...
+        ["chan message 1-1"],
+      );
+    });
+    Rhum.testCase("chan2 should have a message", async () => {
+      socketServer.openChannel("chan2");
+      socketServer.on("chan2", () => {
+        socketServer.on("chan2", (packet: Packet) => {
+          storage["chan2"].messages.push(packet.message);
+        });
+      });
```
Comment (ebebbington) [inline tests/integration/app_3000/tests.ts:96]: Was just curious why there's 2 `on`'s for the same channel?

File: tests/integration/app_3001/tests.ts [hunk 2]
```diff
@@ -35,120 +27,110 @@ const webServer = new Drash.Http.Server({
...
+        ["chan message 1-1"],
+      );
+    });
+    Rhum.testCase("chan2 should have a message", async () => {
+      socketServer.openChannel("chan2");
+      socketServer.on("chan2", () => {
+        socketServer.on("chan2", (packet: Packet) => {
+          storage["chan2"].messages.push(packet.message);
+        });
+      });
```
Comment (ebebbington) [inline tests/integration/app_3001/tests.ts:98]: Same as above comment

File: tests/integration/app_3000/tests.ts (hunk 1 above)
Comment (crookse) [reply tests/integration/app_3000/tests.ts:96]: woops LOL

File: tests/integration/app_3001/tests.ts (hunk 2 above)
Comment (crookse) [reply tests/integration/app_3001/tests.ts:98]: same as above comment

Comment (crookse) [reply tests/integration/app_3001/tests.ts]: b57aa3a

File: tests/integration/app_3000/tests.ts (hunk 1 above)
Comment (crookse) [reply tests/integration/app_3000/tests.ts]: b57aa3a

**Pull request merged**